code/modules/surgery/bodyparts/species_parts/vox_bodyparts.dm

```diff
@@ -0,0 +1,45 @@
+/obj/item/bodypart/head/vox
```

Comment:
We don’t have Vox?

Reply:
Whoops. Forgot to delete that from when I copy pasted the entirety of that folder.